homeassistant/components/rfxtrx/__init__.py

```diff
@@ -70,9 +72,11 @@
 
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
-        vol.Required(CONF_DEVICE): cv.string,
+        vol.Exclusive(CONF_DEVICE, CONNECTION_GROUP): cv.string,
+        vol.Exclusive(CONF_HOST, CONNECTION_GROUP): cv.string,
+        vol.Exclusive(CONF_DUMMY, CONNECTION_GROUP): cv.boolean,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.positive_int,
```

Comment:
port and host should be in a Inclusive group. That mean both are required. There is no point in a default port since this is all custom fetures anyway.

Reply:
I tried but failed. Do you have an example of such a inclusive group within a exclusive group? 

Or is the yaml below fine?  (where i wrap two Required into one Exclusive)

```yaml
rfxtrx:
  network:
    host: 1.2.3.4
    port: 5678
```

Reply:
@elupus do you have an example of such a inclusive group within a exclusive group? Could not find a component/platform that does this.